application/utils.py

```
@@ -91,6 +101,8 @@ def get_headers_dict_from_str(headers_str):
         header_name, header_value = line.split(': ', 1)
         headers[header_name] = header_value
 
+    logger.debug(f'Extracted HTTP headers, headers = %s', headers)
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)